news/2020-04-05-project-loved-april-2020.md

```diff
@@ -0,0 +1,194 @@
+
+[![](/wiki/shared/news/2020-04-05-project-loved-april-2020/mania/2-der-wald-kors-k-remix.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Eleharmonica remixed by kors k - Der Wald (kors k Remix)](https://osu.ppy.sh/beatmapsets/482832#mania) by [snoverpk](https://osu.ppy.sh/users/3595196)  
+170 BPM, 1:59 | \[4K\] 6.03★
+
+snoverpk, as many high level players know, has his fair share of extremely difficult charts, like *Angel Dust* and *EXILE*, made to test the very limits of the capabilities of the best of the best StepMania and osu!mania players. *Der Wald* is a prime example of one of these charts, with some of the fastest light jumpstreams in the game, clocking in at a blistering 340 BPM, with some small snap changes to throw it off just a bit. This is all without mentioning the brutal patterning in these streams along with the middle section containing a couple of *510 BPM* bursts, blowing almost any other chart in the game out of the water in terms of pure speed and difficulty. This extreme speed that is tested in this chart has made many a top player flock to it, being very commonly used as a practice chart for the very best. This makes it notorious and very well-known through the community, and makes it very deserving of a Loved nomination.
+
+[![](/wiki/shared/news/2020-04-05-project-loved-april-2020/mania/3-jumpstream-of-fighters-vol-2b.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[(Various Artists) - Jumpstream Of Fighters Vol. 2B](https://osu.ppy.sh/beatmapsets/547215#mania) by [Leo137](https://osu.ppy.sh/users/1177817), [Manwon](https://osu.ppy.sh/users/4129020), [swiftwinter](https://osu.ppy.sh/users/3453387), [AZVW_2](https://osu.ppy.sh/users/6328227) and [blazziken](https://osu.ppy.sh/users/2544313)  
+200 BPM, 14:19 | \[4K\], 4.74★ – 6.02★  
+The \[(Marathon) - Delete Upon Download\] difficulty is *not* being nominated for Loved.
+
+There's some more StepMania coming your way with about half of a very popular pack released back in late 2016, being the *Jumpstream Of Fighters 2* pack! As the name is likely to tell you, the pack is themed around a range of jumpstream difficulties. This is not all that the pack has to offer, including some easier jack-based gameplay in the Medium chart for *satori de pon!* and the dump streams and tech found in *Live Nor Die*. With many of the charts having relatively simplistic structure and straightforward gameplay, the pack gained immense popularity among the StepMania players it was released for at the time, and of course has maintained a level of recognition and popularity in osu!mania for similar reasons. This is a callback to some of the charting styles used a few years ago, and a shoe-in for the Loved section.
```

Comment:
```suggestion
There's some more StepMania coming your way with about half of a very popular pack released back in late 2016, being the *Jumpstream Of Fighters 2* pack! As the name is likely to tell you, the pack is themed around a range of jumpstream difficulties. This is not all that the pack has to offer, including some easier jack-based gameplay in the Medium chart for *satori de pon!* and the dump streams and tech found in *Live Nor Die*. With many of the charts having relatively simplistic structure and straightforward gameplay, the pack gained immense popularity among the StepMania players it was released for at the time, and of course, has maintained a level of recognition and popularity in osu!mania for similar reasons. This is a callback to some of the charting styles used a few years ago, and a shoo-in for the Loved section.
```

Reply:
never knew the correct spelling of this apparently 😅